templates/events-worker-deployment.yaml

```diff
@@ -116,5 +116,9 @@ spec:
           livenessProbe:
             exec:
               command: ["/bin/bash", "-l", "-c", "bundle exec sidekiqmon processes | grep $(hostname) || exit 1"]
+            initialDelaySeconds: {{ .Values.eventsWorker.livenessProbe.initialDelaySeconds }}
+            periodSeconds: {{ .Values.eventsWorker.livenessProbe.periodSeconds }}
+            timeoutSeconds: {{ .Values.eventsWorker.livenessProbe.timeoutSeconds }}
+            failureThreshold: {{ .Values.eventsWorker.livenessProbe.failureThreshold }}
```

Comment:
Hi @zhaoyue722 whould it be possible to have default case to avoid breaking changes ? Something like :

```
initialDelaySeconds: {{ .Values.worker.livenessProbe.initialDelaySeconds | default 0 }}
periodSeconds: {{ .Values.worker.livenessProbe.periodSeconds | default 10 }}
timeoutSeconds: {{ .Values.worker.livenessProbe.timeoutSeconds | default 1 }}
failureThreshold: {{ .Values.worker.livenessProbe.failureThreshold | default 3 }}
```

Reply:
comments addressed